code/game/jobs/job/imperium.dm

```diff
@@ -0,0 +1,152 @@
+/datum/job/imperial
```

Comment:
Make it follow the other files organization please:
job
outfit
job
outfit

instead of
job
job
outfit 
outfit

Reply:
k